File: Algorithm/src/Algorithm/Python/programmers/monthCodeTest/P68935.py
def solution(n):
    answer = 0
    l = []
    while n > 0:
        l.append(n % 3)
        n //= 3
    j = len(l)
    for i in range(len(l)):
        answer += l[i] * (3 ** (j - 1))
        j -= 1
    return answer


print(solution(125))

File: Algorithm/src/Algorithm/Java/programmers/MonthCodeTest/P68935.java
package Algorithm.Java.programmers.MonthCodeTest;

public class P68935 {
    public static void main(String[] args) {
        System.out.println(solution(125));
    }

    public static int solution(int n) {
        int answer = 0;
        String str = "";
        while (n > 0) {
            str += ""+(n%3);
            n /= 3;
        }
        int cnt = 1;
        for (int i = str.length()-1; i >= 0; i--){
            int c = Integer.parseInt(str.charAt(i)+"");
            if (i == str.length()-1) {
                answer += c;
                continue;
            }
            answer += Math.pow(3, cnt) * c;
            cnt++;
        }

        return answer;
    }
}
